### PR TITLE
Cu 59x1dd clark chen

### PR DESCRIPTION
**Pull request by Clark-spec**
lottery draw show history

### CODE REVIEW

**Comment by Clark-spec** [comment]
update

**Comment by Clark-spec** [comment]
update

File: lottery-clientapi/src/main/java/com/arbfintech/station/lottery/clientapi/controller/api/LotteryPoolApiController.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    @PostMapping("/add-owner")
+    public String addLotteryPoolOwner(
+            @RequestParam Long lotteryHolderId,
+            @RequestParam Integer num
+    ) {
+        LOGGER.info("Start to add lottery or update lottery pool");
+        return lotteryPoolService.addOwnerAndStatusByNum(lotteryHolderId, num);
+    }
+
+    @DeleteMapping("/delete")
```
Comment (roushanyyzz) [inline lottery-clientapi/src/main/java/com/arbfintech/station/lottery/clientapi/controller/api/LotteryPoolApiController.java:29]: update the url and function name

**Review by roushanyyzz — CHANGES_REQUESTED** [review]
Update

File: lottery-object/pom.xml [hunk 2]
```diff
@@ -98,6 +103,10 @@
             <artifactId>flying-saucer-pdf-itext5</artifactId>
             <version>9.1.5</version>
         </dependency>
+        <dependency>
```
Comment (roushanyyzz) [inline lottery-object/pom.xml:106]: What does the dependency do?

File: lottery-object/pom.xml [hunk 3]
```diff
@@ -24,6 +24,11 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
```
Comment (roushanyyzz) [inline lottery-object/pom.xml:27]: Remove all maven that not use

File: lottery-domain/src/main/java/com/arbfintech/station/lottery/domain/entity/LotteryPool.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+public class LotteryPool {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column
+    private String owner;
+
+    @Column(length = 5)
```
Comment (roushanyyzz) [inline lottery-domain/src/main/java/com/arbfintech/station/lottery/domain/entity/LotteryPool.java:15]: Don't limit the length

**Comment by Clark-spec** [comment]
update

**Review by roushanyyzz — APPROVED** [review]

**Pull request merged**